### DISCUSSION TITLE
How to view clips from outside my home network?

### DISCUSSION THREAD

**Discussion by KevinTyrrell** [Hubro/clipface#66] [Q&A]
I've installed Docker and Clipface. Upon typing in my local IP address `10.0.0.151` and port `80`, I'm able to view my clips at the address `10.0.0.151:80`. I can view and play them from any device as long as I use that URL.

However how do I link my clips to others who are **not** on my home network? I see from the readme there are ways to have a url that begins with `https://clipface.com`, however I cannot find any specific directions on how to accomplish this. Going to `https://clipface.com` yields a `Problem loading webpage` error after timing out.

I was able to port forward port `80`, then via [whatsmyip.org](https://www.whatsmyip.org/), connect to it via `my.public.ip.address.here:80`. However this is going to scare people when I send them a direct IP address. There must be a cleaner way to do this, e.g. `https://clipsafe.com/user/KevinTyrrell/` or something.

**Comment by Hubro** [comment]
Sounds like you just need to buy a domain name and point it to your IP address :slightly_smiling_face: Use any domain name registrar (Google Domains, GoDaddy, Namecheat etc.) and point the A record of the domain to your IP address.

Clipface is entirely an open source, self hosted project, I don't offer any services. What you see on the front page is an example of a NGINX reverse-proxy config with SSL enabled, for those who want to use HTTPS. If you just want to share clips with friends then HTTPS is probably unnecessary.
